run.py

```diff
@@ -52,8 +43,11 @@
 # """
 
 ##############################################################################
+RUN_DEVICE = 'gpu' # gpu 或 dml 或 cpu
+MODEL_NAME = './model/%s' % DATA_NAME # 模型名
```

Comment:
```suggestion
MODEL_NAME = './model/{}'.format(DATA_NAME) # 模型名
```

用 `%` 格式化字符串 python 官方已经不推荐了，使用 `format` 吧。上面的建议代码应该是对的，如果不对自己调整下。

Reply:
收到